packages/svelte2tsx/package.json

```diff
@@ -25,13 +29,10 @@
         "mocha": "^6.2.2",
         "periscopic": "^2.0.2",
         "rollup": "^2.28.0",
-        "@rollup/plugin-commonjs": "^15.0.0",
         "rollup-plugin-delete": "^2.0.0",
-        "@rollup/plugin-json": "^4.0.0",
-        "@rollup/plugin-node-resolve": "^9.0.0",
-        "@rollup/plugin-typescript": "^6.0.0",
         "source-map": "^0.6.1",
         "source-map-support": "^0.5.16",
+        "sucrase": "^3.17.0",
```

Comment:
Could we use `ts-node` instread? We already have that in this repo and it might serve the same use case and we avoid another dependency.

Reply:
I agree and I tried many things, but ts-node refuses to work due to a side issue in tsconfig
Sucrase just works, it's also way faster